packages/nimble-components/src/select/styles.ts

```diff
@@ -28,6 +40,88 @@ export const styles = css`
     [part='end'] {
         display: contents;
     }
+
+    .listbox {
+        overflow-x: clip;
+    }
+
+    .listbox.empty slot {
+        display: none;
+    }
+
+    .listbox.above {
+        flex-flow: column-reverse;
+    }
+
+    .filter-field {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        height: ${controlHeight};
+        background: transparent;
+    }
+
+    .filter-icon {
+        padding-left: ${smallPadding};
+    }
+
+    .filter-field::after,
+    .filter-field::before {
+        content: '';
+        position: absolute;
+        height: 0px;
+        border-bottom: rgba(${borderRgbPartialColor}, 0.1) 2px solid;
+    }
+
+    .filter-field::after {
+        top: calc(${controlHeight} + ${smallPadding} - ${borderWidth});
+    }
+
+    .filter-field:before {
+        bottom: calc(${controlHeight} + ${smallPadding});
+    }
+
+    .filter-field:not(.above)::after,
+    .filter-field.above::before {
+        width: calc(100% - (2 * ${borderWidth}));
+    }
+
+    .filter-input {
+        background: transparent;
+        border: none;
+        color: inherit;
+        font: inherit;
+        height: var(--ni-nimble-control-height);
+        padding: 0 ${smallPadding} 0 ${mediumPadding};
+        width: 100%;
+    }
+
+    .filter-input::placeholder {
+        color: ${placeholderFontColor};
+    }
+
+    .filter-input${focusVisible} {
+        outline: 0px;
+    }
+
+    .scrollable-element {
+        overflow: auto;
+    }
+
+    .no-results-label {
+        color: ${placeholderFontColor};
+        height: ${controlHeight};
+        display: inline-flex;
+        align-items: center;
+    }
+
+    .no-results-label.below {
+        padding: ${smallPadding} ${mediumPadding} 0 ${mediumPadding};
+    }
+
+    .no-results-label.above {
+        padding: 0 ${mediumPadding} ${smallPadding} ${mediumPadding};
+    }
```

Comment:
Why aren't these the same styling of `padding: ${smallPadding} ${mediumPadding};`? Perhaps I'm mistaken, but I thought one of the goals was for the height of the drop-down to be the same with the "No items found" text as with a single item. If that's the case, then don't you need the 4px padding on the top & bottom of this span?